vibi-dpu/src/db/github/auth.rs

```diff
@@ -1,15 +1,10 @@
-use std::time::SystemTime;
-use std::time::UNIX_EPOCH;
 use sled::IVec;
 
 use crate::db::config::get_db;
 use crate::utils::github_auth_info::GithubAuthInfo;
 
 pub fn save_github_auth_info_to_db(auth_info: &mut GithubAuthInfo) {
     let db = get_db();
-    let now = SystemTime::now();
-    let since_epoch = now.duration_since(UNIX_EPOCH).expect("Time went backwards");  
-    auth_info.set_timestamp(since_epoch.as_secs());
```

Comment:
I didn't quite understand why this change was there in the first place (why would we change the timestamp field to duration?).